### DISCUSSION TITLE
blog/autocomplete-is-not-all-you-need

### DISCUSSION THREAD

**Discussion by giscus** [zackproser/portfolio#600] [Announcements]
# blog/autocomplete-is-not-all-you-need

AI assisted developer tooling is not created equally...

https://zackproser.com/blog/autocomplete-is-not-all-you-need

**Comment by CallMePlayer** [comment]
Cursor has been fun for me.
"I suspect this year I'll be able to build an app by speaking into my phone." 
Yeah, that's my goal too.

**Reply by zackproser** [reply]
Nice, I think it would be an ideal UX improvement since I tend to have better ideas once I've been hacking for a while and then step away to do something else like walk the dog...
